release-notes/dashboards-observability.release-notes-2.11.0.0.md

```diff
@@ -0,0 +1,43 @@
+## Version 2.11.0.0 Release Notes
+
+Compatible with OpenSearch and OpenSearch Dashboards Version 2.11.0
+
+### Features
+* Add nightly cron schedule to E2E workflow in https://github.com/opensearch-project/dashboards-observability/pull/1005
```

Comment:
We should updates this to the below: https://github.com/opensearch-project/opensearch-plugins/blob/main/RELEASE_NOTES.md
```
Add nightly cron schedule to E2E workflow  ([#1005](https://github.com/opensearch-project/dashboards-observability/pull/1005))